layouts/community/ortho_4x12/symbolic/keymap.c

```diff
@@ -0,0 +1,173 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
```

Comment:
These two includes aren't actually required.  You can safely remove them, ans the `QMK_KEYBOARD_H` include should actually cover them.